samples/client/petstore/dart/flutter_petstore/openapi/lib/model/api_response.dart

```diff
@@ -2,11 +2,11 @@ part of openapi.api;
 
 class ApiResponse {
   
-  int code = null;
+  int code = ;
```

Comment:
I think there might be something wrong

Reply:
template needs updatinb

Reply:
I started looking into it, but decided this is too far from scope of this MR. Might look into it later, for now I reverted the change for dio and jaguar.